ecs-logs-collector.sh

```diff
@@ -419,19 +418,20 @@ get_docker_info()
 {
   try "gather Docker daemon information"
 
-  pgrep docker > /dev/null
+  ps -ef |grep dockerd |grep -v grep  >> /dev/null
```

Comment:
Good catch! This catches the `dockerd` process rather than any processes with `docker` in the name. This line is a bit noisy however and could be `pgrep dockerd` to clean it up a bit.